frontend-ui/src/components/Login.js

```diff
@@ -0,0 +1,126 @@
+import React, {useEffect, useState} from "react";
+import axios from 'axios';
+import {useHistory} from "react-router-dom";
+
+function LoginNavbar() {
+    return (
+        <div className={"gray-navbar"}>
+            <p className={"left-item"}>LOGIN </p>
+        </div>
+    );
+}
+const Login = () => {
+    let history = useHistory();
+    const [credentials, setCredentials] = useState({
+        email: '',
+        password: '',
+        rememberMe: ''
+    })
+    useEffect(() => {
+        if (localStorage.checkbox && localStorage.checkbox !== "") {
+            setCredentials({
+                ...credentials,
+                ...{
+                    email: localStorage.email,
+                    rememberMe: localStorage.checkbox
+                }
+            })
+        } else {
+            setCredentials({
+                ...credentials,
+                ...{
+                    email: "",
+                    rememberMe: ""
+                }
+            })
+        }
+    }, []);
+    const lsRememberMe = () => {
+        if (credentials.rememberMe && credentials.email !== "") {
+            localStorage.email = credentials.email;
+            localStorage.checkbox = credentials.rememberMe;
+        } else {
+            localStorage.email = "";
+            localStorage.checkbox = "";
+        }
+    }
+    const handleEmailChange = (e) => {
+        setCredentials({
+            ...credentials,
+            ...{email: e.target.value.toString()}
+        })
+    }
+    const handleRememberMeChange = (e) => {
+        setCredentials({
+            ...credentials,
+            ...{rememberMe: e.target.checked}
+        })
+    }
+    const handlePasswordChange = (e) => {
+        setCredentials({
+            ...credentials,
+            ...{password: e.target.value.toString()}
+        })
+    }
+    const handleLoginClick = async (e) => {
+        lsRememberMe();
+        const url = "http://localhost:8080/api/login"
+        const config = {
+            headers: {
+                'Content-Type': 'application/json',
+                'Access-Control-Allow-Origin': '*',
+                'Access-Control-Allow-Methods': 'GET,PUT,POST,DELETE,PATCH,OPTIONS'
+            }
+        }
+        try {
```

Comment:
You should extract methods like this (methods related to api calls) in separate files (e.g. api/auth.js). In this way, you will have similar methods grouped and could reuse them in other components inside application. Also  react components files will be cleaner and easier to read.